tests/data/rsa2-private.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Is this file used? I don't see it referenced in this diff.

Reply:
It's not used directly at the moment. But as the fake JSON response is containing a key set with two public keys I thought it's helpful to also include both private keys... Should I remove it?

Reply:
No that's ok.